Say Their Names/Source/Controller/Home/Carousel/Collectionview/ResultsDataHandler.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ResultsDataHandler.swift
+//  Say Their Names
+//
+//  Created by Thomas Murray on 03/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+
+///Provides default methods for interfacing with the results handler class.
+protocol DataHandlerProtocol {
+    var resultsHandler: ResultsDataHandler {get set}
+    var resultsData: [Any]? {get}
```

Comment:
I've added an associated type into the protocol for use rather than using Any. It's not something I've used, but it seems to fit the bill nicely. Thanks for that.